region/maridia/inner-pink.json

```diff
@@ -3303,7 +3321,6 @@
                     ]},
                     {"canShineCharge": {
```

Comment:
I know this is out of scope, but can this be shinecharge be deleted? If not, should it be moved into the `canStutterWaterShineCharge` strat?

Reply:
The `canShineCharge` doesn't actually mean anything logically.  A shinecharge is still be performed with either `canWaterShineCharge` or `canStutterWaterShineCharge`.  And those techs are together with the needed requirements for a shineCharge to work, so we can say that there is a shineCharge occurring.  And that makes the `shinespark` in the other strats not out of place, since there is still a shineCharge here.

This `canShineCharge` should be able to be deleted.